src/sentry/mail/adapter.py

```diff
@@ -188,61 +193,72 @@ def get_send_to(self, project, target_type, target_identifier=None, event=None):
             logger.debug("Tried to send notification to invalid project: %r", project)
             return set()
 
-        send_to = []
         if target_type == ActionTargetType.ISSUE_OWNERS:
             if not event:
-                send_to = self.get_send_to_all_in_project(project)
+                return self.get_send_to_all_in_project(project)
             else:
-                send_to = self.get_send_to_owners(event, project)
+                return self.get_send_to_owners(event, project)
         elif target_type == ActionTargetType.MEMBER:
-            send_to = self.get_send_to_member(project, target_identifier)
+            # TODO(ceo): this is set to just email for now, but when we update the alert rule UI
+            # to allow you to choose "notification" rather than "email" this will need to change
+            return {ExternalProviders.EMAIL: self.get_send_to_member(project, target_identifier)}
```

Comment:
I know I commented it but I really wanna call out that this (and line 206) are what will need to be changed when we have the front end.